core/server/middleware/error-handler.js

```diff
@@ -4,9 +4,11 @@ var _ = require('lodash'),
     config = require('../config'),
     errors = require('../errors'),
     i18n = require('../i18n'),
-    _private = {};
+    _private = {},
+    errorHandler = {},
+    htmlStackExclusions = ['NotFoundError', 'DatabaseNotPopulatedError'];
 
-_private.parseStack = function (stack) {
+_private.parseStack = function parseStack(stack) {
```

Comment:
Is that instead of https://github.com/TryGhost/Ghost/pull/7548/files/79e7eba9fb0c2d7dcc4b4afb48beafc952f4f693#diff-6b41d6f0ef148f7d202931a87d5c4130R118?